src/collections/Breadcrumb/Breadcrumb.js

```diff
@@ -25,28 +25,23 @@ function Breadcrumb(props) {
     return <div className={classes} {...rest}>{children}</div>
   }
 
-  let dividerJSX = <Breadcrumb.Divider />
+  const dividerJSX = <Breadcrumb.Divider icon={icon}>{divider}</Breadcrumb.Divider>
+  const sectionsJSX = []
 
-  if (divider) {
-    dividerJSX = <Breadcrumb.Divider>{divider}</Breadcrumb.Divider>
-  }
+  sections.forEach(({ text, ...restSection }, index) => {
+    const key = `${text}-${index}`
+    const dividerKey = `$divider-${index}`
 
-  if (icon) {
-    dividerJSX = <Breadcrumb.Divider icon={icon} />
-  }
+    sectionsJSX.push(
+      <Breadcrumb.Section {...restSection} key={key}>{text}</Breadcrumb.Section>
+    )
 
-  return (
-    <div className={classes} {...rest}>
-      {sections.map((section, index) => {
-        const isLast = index === (sections.length - 1)
-
-        return [
-          <Breadcrumb.Section {...section}>{section.text}</Breadcrumb.Section>,
-          isLast ? null : dividerJSX,
-        ]
-      })}
-    </div>
-  )
+    if (index !== sections.length - 1) {
+      sectionsJSX.push(React.cloneElement(dividerJSX, { key: dividerKey }))
```

Comment:
You proposed
```js
sectionsJSX.push(React.cloneElement(dividerJSX, { key }))
```

But it seems incorrect (dividers are not rendered), I added another key for divider. It's correct?